plugins/backfill/transformerExporter.go

```diff
@@ -13,7 +24,19 @@ type exporter string
 var Exporter exporter
 
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
-	return []event.TransformerInitializer{},
+	return []event.TransformerInitializer{
+			clip_kick.EventTransformerInitializer,
+			clip_redo.EventTransformerInitializer,
+			clip_take.EventTransformerInitializer,
+			clip_yank.EventTransformerInitializer,
+			dog_deny.EventTransformerInitializer,
+			dog_digs.EventTransformerInitializer,
+			dog_rely.EventTransformerInitializer,
+			dog_file_ilk_clip.EventTransformerInitializer,
+			dog_file_hole.EventTransformerInitializer,
+			dog_file_ilk_chop_hole.EventTransformerInitializer,
```

Comment:
Weird formatting here...

Reply:
yeah.. strange. In goland its lined up under the `[ ]`. I'll see if its my tab setting.